File: exs/mundo_1/java/030.java
/*
    ### 030 - Par ou ímpar?
    Crie um programa que leia um número inteiro e mostre na tela se ele é PAR ou ÍMPAR
 */


import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        int numero;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Digite um numero: ");
        numero = scanner.nextInt();

        if (numero % 2 == 0)
        {
            System.out.println("É par");
        }
        else
        {
            System.out.println("É impar");
        }
    }
}

File: exs/mundo_1/c++/030.cpp
/*

030 - Par ou ímpar?
	Crie um programa que leia um número inteiro e
	mostre na tela se ele é PAR ou ÍMPAR

*/

#include <iostream>

using namespace std;

int main() {
	int n;

	cin >> n;
	if (n % 2 == 0) {
		cout << "É par\n";
	} else {
		cout << "É impar\n";
	}

	return 0;
}
